apps/src/javalab/CommitDialog.jsx

```diff
@@ -3,35 +3,50 @@ import PropTypes from 'prop-types';
 import i18n from '@cdo/javalab/locale';
 import color from '@cdo/apps/util/color';
 import {CompileStatus} from './constants';
-import StylizedBaseDialog from '@cdo/apps/componentLibrary/StylizedBaseDialog';
+import StylizedBaseDialog, {
+  FooterButton
+} from '@cdo/apps/componentLibrary/StylizedBaseDialog';
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
 
 export default class CommitDialog extends React.Component {
   state = {
     filesToCommit: [],
+    filesToBackpack: [],
     commitNotes: null
   };
 
   renderFooter = buttons => {
     let compileStatusContent = '';
-    switch (this.props.compileStatus) {
-      case CompileStatus.LOADING:
-        compileStatusContent = i18n.compiling();
-        break;
-      case CompileStatus.SUCCESS:
-        compileStatusContent = [
-          <FontAwesome
-            key="icon"
-            icon="check-circle"
-            className="fa-2x"
-            style={styles.iconSuccess}
-          />,
-          <span key="text">{i18n.allFilesCompile()}</span>
-        ];
-        break;
-      case CompileStatus.ERROR:
-        compileStatusContent = i18n.compileFailed();
-        break;
+    let commitText = 'Commit';
```

Comment:
should this (and 'Cancel' below) be translated?

Reply:
yes! done:)